src/chainlit/frontend/src/components/organisms/FormInput.tsx

```diff
@@ -0,0 +1,60 @@
+import Switch, { SwitchProps } from 'components/atoms/switch';
+
+import { IInput } from 'types/Input';
+
+import SelectInput, {
+  SelectInputProps,
+  SelectItem
+} from './inputs/selectInput';
+import TagsInput, { TagsInputProps } from './inputs/tagsInput';
+import TextInput, { TextInputProps } from './inputs/textInput';
+import Slider, { SliderProps } from './slider';
+
+interface IFormInput<T, I> extends IInput {
+  type: T;
+  initial: I;
+}
+
+export type TFormInput =
+  | (SwitchProps & IFormInput<'switch', boolean>)
+  | (SliderProps & IFormInput<'slider', number>)
+  | (SelectInputProps & IFormInput<'select', string>)
+  | (TextInputProps & IFormInput<'textinput', string>)
+  | (TagsInputProps & IFormInput<'tags', string>);
+
+const FormInput = ({ element }: { element: TFormInput }): JSX.Element => {
+  const renderInput = () => {
+    switch (element.type) {
+      case 'select':
+        return (
+          <SelectInput
+            {...element}
+            items={element?.items?.map((option: SelectItem) => ({
```

Comment:
Is the element ever going to be null here?

Reply:
No, even the items should not be null if we use Select